### PR TITLE
Milestone 1.4.3

### PR DESCRIPTION
**Pull request by Scrum**
## Proposed Changes

https://github.com/posthtml/posthtml-expressions/issues/82

## Types of Changes

What types of changes does your code introduce
_Put an `x` in the boxes that apply_

- [x] Bug (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature which changes existing functionality)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating
the PR. If you're unsure about any of them, don't hesitate to ask. We're here to
help! This is a reminder of what we are going to look for before merging your
code._

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) guide
- [x] Lint and unit tests pass with my changes
- [x] I have added tests that prove my fix is effective/works
- [ ] I have added necessary documentation (if appropriate)
- [x] Any dependent changes are merged and published in downstream modules

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -170,10 +172,10 @@ function walk (opts, nodes) {
 
     // don't parse ignoredTag
     if (node.tag === ignored) {
-      node.tag = false
+      // node.tag = false
```
Comment (cossssmin) [inline lib/index.js:175]: Should we just delete this commented out line?

File: test/test-core.js [hunk 2]
```diff
@@ -78,6 +78,10 @@ test('Raw output', (t) => {
   return process(t, 'raw', { locals: { foo: 'bar' } })
 })
 
+test.only('Raw output - inside condition', (t) => {
```
Comment (cossssmin) [inline test/test-core.js:81]: Remove `.only`

**Comment by cossssmin** [comment]
Just realized that this won't work:

```html
<if condition="foo === 'bar'">
  <raw>
    {{ foo }}
  </raw>
  <p class="@{{ dynamicClass }}">{{ foo }}</p>
</if>
```

It does not ignore `@{{ dynamicClass }}` and results in `<p class="undefined">bar</p>`.  Any ideas?

**Comment by cossssmin** [comment]
Added a test for `<raw>` in the `<switch>` tag, since that was problematic as well. Not sure what to do with the issue from my comment above, but I am ok with merging/releasing 1.4.3 and handling it later. 

We could actually tackle #88 right after this, I already fixed the issue locally and am waiting on this one to be merged first. Or, should I make #88 part of this milestone, what do you prefer?

**Comment by Scrum** [comment]
> We could actually tackle #88 right after this, I already fixed the issue locally and am waiting on this one to be merged first. Or, should I make #88 part of this milestone, what do you prefer?

Without a doubt it is better to take after merge this branch

**Comment by Scrum** [comment]
@cossssmin publish v1.4.3

**Pull request merged**